client/Lykke.Service.BlockchainSettings.Client/CacheInvalidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Lykke.Service.BlockchainSettings.Client.Attributes;
+
+namespace Lykke.Service.BlockchainSettings.Client
+{
+    public class CacheInvalidator
+    {
+        private readonly object _locker;
+        private readonly IBlockchainSettingsClient _client;
+        private IEnumerable<(MethodInfo, InvalidateableClientCachingAttribute)> _methodAttributeTuples;
```

Comment:
Make it IReadOnlyCollection to exclude multiple sequence evaluations on Invalidate() call

Reply:
It could be collection just of InvalidateableClientCachingAttribute Item1 of tuple is never used

Reply:
deleted)